src/Symfony/Component/Security/Http/Authenticator/Passport/Policy/PolicyInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http\Authenticator\Passport\Policy;
+
+use Symfony\Component\Security\Core\Exception\AuthenticationException;
+
+interface PolicyInterface
```

Comment:
I'd name this one PasswordPolicyInterface, that's what we use everywhere else in the PR

Reply:
Correct. Done